jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -1252,7 +1252,7 @@ public String getRemoteUser()
     public RequestDispatcher getRequestDispatcher(String path)
     {
         // path is encoded, potentially with query
-
+        // TODO: why do we do this?
         path = URIUtil.compactPath(path);
```

Comment:
Let's try commenting it out for now and seeing what fails
```suggestion
        // TODO: why do did we do this?
        // path = URIUtil.compactPath(path);
```